lib/unparser/emitter/primitive.rb

```diff
@@ -20,6 +20,23 @@ def dispatch
 
       end # Inspect
 
+      class Symbol < self
+
+        handle :sym
+
+        private
+
+        # mutant:disable
+        def dispatch
+          if RUBY_VERSION < '3.2' && value.match?(/=\z/)
```

Comment:
@prikha I'm well aware I probably did this wrong within unparser at other places, but I think we should use: `RUBY_VERSION < '3.2.'` note the extra `.` If ruby ever where to get a minor version of like: `3.11` it would despite logically being bigger, be lexicographical sorted wrong.

Reply:
Next question, how do we know that only `=` suffix symbols are affected? 

I wounder if we should instead do something "stupid" and on `RUBY < '3.2.'` do a round trip on `#inspect` and if that fails add the quotes, and raise if that also fails.

Reply:
Let's go with a single roundtrip. Second one seems to be an overkill to me.

Reply:
Applied